framework-plugins/lisk-framework-forger-plugin/src/forger_plugin.ts

```diff
@@ -187,20 +188,22 @@ export class ForgerPlugin extends BasePlugin {
 			header: { height: lastBlockHeight },
 		} = this.codec.decodeBlock(await this._channel.invoke<string>('app:getLastBlock'));
 		const { syncUptoHeight } = await getForgerSyncInfo(this._forgerPluginDB);
+		const { genesisHeight } = await this._channel.invoke<NodeInfo>('app:getNodeInfo');
+		const forgerPluginSyncedHeight = syncUptoHeight === 0 ? genesisHeight : syncUptoHeight;
```

Comment:
in case of migration the height of regenesis is not 0?

Reply:
`syncUpToHeight === 0` means there is no data in this plugin or actually synced height `0`. therefore, replacing with genesis height should be ok